Mathlib/RingTheory/Ideal/Operations.lean

```diff
@@ -811,6 +811,23 @@ variable {I J} in
 theorem IsRadical.inf (hI : IsRadical I) (hJ : IsRadical J) : IsRadical (I ⊓ J) := by
   rw [IsRadical, radical_inf]; exact inf_le_inf hI hJ
 
+open Finset in
+lemma radical_finset_inf {ι} {s : Finset ι} {f : ι → Ideal R} {i : ι} (hi : i ∈ s)
+    (hs : ∀ ⦃y⦄, y ∈ s → (f y).radical = (f i).radical) :
+    (s.inf f).radical = (f i).radical := by
+  classical
```

Comment:
We should probably show that `Ideal.Radical` is a `InfTopHom` and use [`map_finset_inf`](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Data/Finset/Lattice.html#map_finset_inf)